spec/resolver-fixtures/invalid-duplicate-annotation-error.yml

```diff
@@ -0,0 +1,8 @@
+policy: |
+  - !host
+    id: test-app
+    annotations:
+      description: value1
+      description: value2
+
+error: "Error at line 4, column 16 in  : Duplicate attribute of type description"
```

Comment:
Do these need to purposed exclude the newline at the end of the file? Or can we add these to each hof the resolver fixture yaml files?